python/lsst/ip/isr/crosstalk.py

```diff
@@ -926,6 +892,13 @@ def run(self,
             Do subtraction in parallel overscan region (only)?
         detectorConfig : `lsst.ip.isr.OverscanDetectorConfig`, optional
             Per-amplifier configs used when parallelOverscanRegion=True.
+        fullAmplifier : `bool`, optional
+            Use full amplifier and not just imaging region.
+        doSqrCrosstalk : `bool` or `None`, optional
+            If this is set to False, override the configuration.
```

Comment:
Why do we need the ability to override the config here?

Reply:
So in the case that we don't have proper gains in the new task, then we can't use the non-linear crosstalk term.  But if you're unhappy with this code then the configuration consistency could be checked in the validation in the new ISR task config, and we could make sure that we set it in the pipeline.  I should probably do that.